### PR TITLE
DOC: Remove duplicated `intersphinx_mapping` key

### PR DESCRIPTION
**Pull request by Zeroto521**
pyepsg this package comes up twice in `intersphinx_mapping`.

See below.

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -376,10 +376,6 @@ def setup(app):
         "https://pandas.pydata.org/pandas-docs/stable/objects.inv",
     ),
     "pyarrow": ("https://arrow.apache.org/docs/", None),
-    "pyepsg": (
-        "https://pyepsg.readthedocs.io/en/stable/",
-        "https://pyepsg.readthedocs.io/en/stable/objects.inv",
-    ),   
     "pyepsg": (
```
Comment (Zeroto521) [inline doc/source/conf.py:379]: pyepsg comes up twice.

File: geopandas/io/tests/test_file.py [hunk 2]
```diff
@@ -412,7 +412,8 @@ def test_infer_zipped_file():
     gdf = read_file(path)
     assert isinstance(gdf, geopandas.GeoDataFrame)
 
-    # Check that it can successfully add a zip scheme to a path that already has a scheme
```
Comment (Zeroto521) [inline geopandas/io/tests/test_file.py:415]: pylint gave an error there https://github.com/geopandas/geopandas/runs/3260865832#step:4:56

Should do this in another PR before?

**Comment by martinfleis** [comment]
@Zeroto521 thanks! can you update your branch to fix the listing issue (I messed it up this morning).

**Review by jorisvandenbossche — APPROVED** [review]
Thanks!
